src/validation/facilities.validation.js

```diff
@@ -0,0 +1,60 @@
+import { check, validationResult } from 'express-validator';
+
+// this transforms the default error message
+const customMessage = validationResult.withDefaults({
+  formatter: (error) => (
+    {
+      field: error.param,
+      message: error.msg
+    }
+  )
+});
+
+
+const validateFacilities = [
+  check('name', 'Kindly Provide a name for your facility')
```

Comment:
Can you ensure name, address, description are strings? Putting a number (note, not in quotes) in there passes through? Address and description should at least have letters or a mixture of numbers and letters

Reply:
Alright. I will Implement this. Thanks